front/app/containers/Admin/pagesAndMenu/containers/CustomPages/CustomPageSettingsForm/index.tsx

```diff
@@ -91,22 +92,46 @@ const CustomPageSettingsForm = ({
       is: (value: ProjectsFilterTypes) => value === 'topics',
       then: array().of(string()).min(1, formatMessage(messages.atLeastOneTag)),
     }),
-    area_id: string().when('projects_filter_type', {
-      is: (value: ProjectsFilterTypes) => value === 'areas',
-      then: string().required(formatMessage(messages.selectAnArea)),
-    }),
+    area_id: string()
+      .nullable()
+      .when('projects_filter_type', {
+        is: (value: ProjectsFilterTypes) => value === 'areas',
+        then: string()
+          .nullable()
+          .required(formatMessage(messages.selectAnArea)),
+      }),
   });
 
   const methods = useForm({
     mode: 'onBlur',
     defaultValues,
     resolver: yupResolver(schema),
   });
+
   const slug = methods.watch('slug');
 
   const onFormSubmit = async (formValues: FormValues) => {
+    let newFormValues = { ...formValues };
+
+    // remove values for filter selections other than the final one
+    // if by areas filter or by no filter, don't send topic_ids
+    if (
```

Comment:
So, we need it e.g. for the case when a page is created with both `area_ids` and `topic_ids`? I'm wondering whether we should better handle it on the BE on the model level for more reliability.

Reply:
Yeah, if you pick both tags and areas on the FE, both are sent to the backend by the form, by default. This removes any unused association IDs (areas if topics and vice-versa, or all ids if "no_filter"). It's a bit clunky but I think straightforward to do here. If it's easy to do on the BE we can do that, but I'm not sure it's necessary